### PR TITLE
Update to align with Softlayer CPI NG

### PR DESCRIPTION
**Pull request by gu-bin**
Update init-softlayer.md and softlayer-cpi.md
Add softlayer-cpi-migration.md

### CODE REVIEW

File: content/init-softlayer.md [hunk 1]
```diff
@@ -34,15 +34,29 @@ VLANs provide the ability to partition devices and subnets on the network. To or
 ---
 ## Step 2: Deploy {: #deploy }
 
-1. Install [CLI v2](cli-v2.md).
+1. Install [BOSH CLI v2](cli-v2.md), make sure to download SoftLayer specific version of bosh-cli [linux](https://bosh-softlayer-artifacts.s3.amazonaws.com/bosh-cli-5.0.1.1-softlayer-linux-amd64) or [mac](https://bosh-softlayer-artifacts.s3.amazonaws.com/bosh-cli-5.0.1.1-softlayer-darwin-amd64).
```
Comment (dpb587-pivotal) [inline content/init-softlayer.md]: I don't think we should be encouraging separate CLIs. For general OSS users, it sounded like they can use persistent IPs within IBM Cloud and have the same workflow as all other IaaSes. Was that the only reason users needed this custom `bosh-cli` binaries?

File: content/softlayer-cpi-migration.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
+For the users using legacy Softlayer CPI who want to move to Softlayer CPI NG, please refer to this doc to do the migration.
```
Comment (dpb587-pivotal) [inline content/softlayer-cpi-migration.md]: I think you'd want to use "SoftLayer" capitalization/branding consistently.

File: content/init-softlayer.md (hunk 1 above)
Comment (gu-bin) [reply content/init-softlayer.md]: As many users may use dynamic IP, we'd like to also support it.

**Comment by dpb587-pivotal** [comment]
Hi - can you remove dynamic IP support and custom CLI references from this document?

Given Portable IPs work and do not require an independently maintained fork and different software from the rest of BOSH ecosystem, PIPs should be the standard solution we recommend for the general OSS community.

If you want to maintain documentation in support of dynamic IPs as a SoftLayer-specific adaptation, please feel free to maintain a doc about it in your internal ecosystem.

**Comment by gu-bin** [comment]
@dpb587-pivotal I've updated the docs as you suggested. Please review again. Thanks.

**Comment by dpb587-pivotal** [comment]
Looks fine - thanks!

**Pull request merged**